### PR TITLE
updated the card elements font size

### PR DESCRIPTION
**Pull request by akshay1502**
updated the font size of the elements according to Figma design.

### CODE REVIEW

**Comment by akshay1502** [comment]
#143

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 To run the project `yarn run dev`.
 
 # Production
-
+
```
Comment (ankushdharkar) [inline README.md:14]: Why?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (akshay1502) [reply README.md:14]: Sorry, I might have accidentally pressed enter or tab while reading the readme.md file.

Comment (ankushdharkar) [reply README.md:14]: Can you fix it please? And request a re-review?

Comment (akshay1502) [reply README.md:14]: Ok

File: src/components/tasks/card/card.module.scss [hunk 2]
```diff
@@ -49,6 +49,6 @@
 }
 
 .cardStrongFont {
-  font-size: 1.5rem;
+  font-size: 1.3rem;
```
Comment (whyDontI) [inline src/components/tasks/card/card.module.scss:52]: Can we remove extra spaces?
Also, make sure that the `rem` unit converts to whatever font-size suggested in the designs

**Review by whyDontI — CHANGES_REQUESTED** [review]
Requesting some linting changes

File: src/components/tasks/card/card.module.scss (hunk 2 above)
Comment (akshay1502) [reply src/components/tasks/card/card.module.scss:52]: ok

**Review by whyDontI — CHANGES_REQUESTED** [review]
@akshay1502 The previous changes were correct, we don't want a fixed font size for all devices.
That's why we use `em` and `rem` units

`em` is relative to the font-size of its direct or nearest parent, `rem` is only relative to the html (root) font-size.

You might want to check this blog out,
https://css-tricks.com/rem-global-em-local/

Good work 🥳 👍🏽

**Comment by swarajpure** [comment]
I was comparing the previous commit to the latest commit and hence I was seeing things like 16px => 1.1rem and hence my comment

My bad.

You can keep the values according to figma design (rem only)

**Review by whyDontI — APPROVED** [review]
Good work, Please take care of the lint issues from next time 😃

**Pull request merged**